### DISCUSSION TITLE
Newbie trying to use llama-cli file mode

### DISCUSSION THREAD

**Discussion by lyh552506** [ggml-org/llama.cpp#12184] [Q&A] [answered]
Hi everyone,

Sorry for the noob question in advance.

I'm tring the `llama-cli` with command (using a qemu-riscv simulator):
`qemu-riscv64 -L /opt/riscv/sysroot/ -cpu rv64,v=true,vlen=256,elen=64,vext_spec=v1.0 ./build/bin/llama-cli -m ../autodl-tmp/DeepSeek-R1-Distill-Qwen-1.5B-Q2_K.gguf --file=prompt.txt -t 8`
in which I want to give all my prompts in `prompt.txt`, and hope the model will answer these one by one

However, it keep reminding me I was still in interactive mode 

`
main: llama threadpool init, n_threads = 8
main: chat template is available, enabling conversation mode (disable it with -no-cnv)
*** User-specified prompt in conversation mode will be ignored, did you mean to set --system-prompt (-sys) instead?
main: chat template example:
You are a helpful assistant
`
`
system_info: n_threads = 8 (n_threads_batch = 8) / 192 | CPU : RISCV_V = 1 | LLAMAFILE = 1 | OPENMP = 1 | AARCH64_REPACK = 1 | 
`
`main: interactive mode on.
sampler seed: 1769281452
sampler params: 
        repeat_last_n = 64, repeat_penalty = 1.000, frequency_penalty = 0.000, presence_penalty = 0.000
        dry_multiplier = 0.000, dry_base = 1.750, dry_allowed_length = 2, dry_penalty_last_n = 4096
        top_k = 40, top_p = 0.950, min_p = 0.050, xtc_probability = 0.000, xtc_threshold = 0.100, typical_p = 1.000, top_n_sigma = -1.000, temp = 0.800
        mirostat = 0, mirostat_lr = 0.100, mirostat_ent = 5.000
sampler chain: logits -> logit-bias -> penalties -> dry -> top-k -> typical -> top-p -> min-p -> xtc -> temp-ext -> dist 
generate: n_ctx = 4096, n_batch = 2048, n_predict = -1, n_keep = 1`

== Running in interactive mode. ==
 - Press Ctrl+C to interject at any time.
 - Press Return to return control to the AI.
 - To return control without starting a new line, end your input with '/'.
 - If you want to submit another line, end your input with '\'.
 - Not using system message. To change it, set a different value via -sys PROMPT

I wonder how to disable interactive mode and use file mode

Thanks for your help!!!!

**Comment by fairydreaming** [comment, marked as answer]
@lyh552506 Use `-no-cnv`, it explicitly disables interactive mode. Few months ago interactive mode was disabled by default, now I think it's is enabled by default, this causes some confusion around.

**Reply by lyh552506** [reply]
Thanks a lot! It really help me !!!

**Comment by lyh552506** [comment]
已收到，谢谢This is an automatic reply, confirming that your e-mail was received.Thank you
